### PR TITLE
Make user guide user-centric

### LINKED ISSUE #237: Make the UG user centric

### PR DESCRIPTION
**Pull request by Respirayson**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -1225,7 +1224,7 @@ This section describes commands tht help you manage your students' graded tests.
 
 #### 🛠️Editing a Graded Test Score: `editgradedtest`
 
-You can edit your student’s graded test scores.
+You can edit your student's graded test scores if there is a need to make adjustments or corrections to the initially recorded test scores.
```
Comment (raydenlim) [inline docs/UserGuide.md]: You can edit your student's graded test scores if `you` need to make adjustments or corrections to their initially recorded test scores.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -1289,7 +1288,7 @@ This section describes commands that help you manage your sessions.
 
 #### 📝Creating a Session: `createsession`
 
-You can create a session for any upcoming or past sessions.
+You can create a session for any upcoming or past sessions if you need to schedule or document class sessions in the system.
```
Comment (raydenlim) [inline docs/UserGuide.md]: You can create a session for any upcoming or past sessions if you need to schedule or document `your` class sessions in the system.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -1335,7 +1334,7 @@ Format: `createsession s/SESSION_NUMBER n/STUDENT_NAME n/STUDENT_NAME …`
 
 #### 🛠️Updating a Session's Remark: `updatesessionremark`
 
-You can update an existing session's remark.
+You can update an existing session's remark if there are changes or additional information that you want to include in the session notes or details.
```
Comment (raydenlim) [inline docs/UserGuide.md]: You can update an existing session's remark if you need to change or add additional information to the session notes or details.

Change it such that the format follows `if you need to ...`

File: docs/UserGuide.md [hunk 4]
```diff
@@ -1479,7 +1478,7 @@ Format: `createconsult d/DATE tt/TIME n/STUDENT_NAME n/STUDENT_NAME …`
 <br>
 
 #### 📝️Adding students to a consultation: `addtoconsult`
-You can add your student(s) into a consultation slot.
+You can add your student(s) into a consultation slot if you want to schedule or allocate specific individuals for the consultation.
```
Comment (raydenlim) [inline docs/UserGuide.md]: You can add your student(s) into a consultation slot if you `need` to schedule or allocate specific individuals for `your` consultation.

File: docs/UserGuide.md [hunk 5]
```diff
@@ -1527,7 +1526,7 @@ Format: `addtoconsult INDEX n/STUDENT_NAME …`
 
 #### ❌Removing Students from a Consultation: `removefromconsult`
 
-You can remove your student(s) from a consultation.
+You can remove your student(s) from a consultation if there is a need to reschedule or cancel their participation in the scheduled consultation.
```
Comment (raydenlim) [inline docs/UserGuide.md]: You can remove your student(s) from a consultation if you need to reschedule or cancel your student's participation in the scheduled consultation.

**Review by raydenlim — APPROVED** [review]
Other than a few inconsistencies, LGTM!

File: docs/UserGuide.md [hunk 6]
```diff
@@ -420,7 +420,7 @@ For the list of tab indexes to use:
 
 #### Viewing help : `help`
 
-You can view a message explaining how to access the help page.
+You can view a message explaining how to access the help page, if you require assistance in understanding the commands and their formats.
```
Comment (ldinghan) [inline docs/UserGuide.md:423]: i think this can do without the comma

**Pull request merged**